src/tests/GC/API/GC/GetTotalPauseDuration.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// Tests GC.GetTotalPauseDuration()
+
+using System;
+using System.Diagnostics;
+
+public class Test_Collect
+{
+    public static int Main()
+    {
+        Stopwatch sw = Stopwatch.StartNew();
+        GC.Collect();
+        sw.Stop();
+        TimeSpan elapsed = sw.Elapsed;
+        TimeSpan totalPauseDuration = GC.GetTotalPauseDuration();
```

Comment:
Looks like this is failing on mono.  You might need to ensure that this test is either building against System.Private.CoreLib or calling this API through reflection.